app/annotation/annotation.cjsx

```diff
@@ -108,10 +111,24 @@ Annotation = React.createClass
       @overlay.classList.remove 'fade-out'
 
       @refs.imageZoom.hide()
+      @setState zoomImageCurrent: null
       @setState zoomImageSrc: null
-      @overlay.removeEventListener 'click'
+      @overlay.removeEventListener 'click', @closeZoom
+      window.removeEventListener 'keydown', @onPressKeyInZoom
     ), 250
 
+  onPressKeyInZoom: (e) ->
+    if e.keyCode is 37 or e.keyCode is 39
+      e.preventDefault()
+
+      i = @state.zoomImageCurrent
+      if e.keyCode is 37
```

Comment:
Currently, I can hit left arrow and get a negative index value. Could this be prevented or looped to the last item in the array so we don't get an undefined image source?